lib/yabeda/sidekiq.rb

```diff
@@ -104,5 +108,23 @@ def custom_tags(worker, job)
         worker.method(:yabeda_tags).arity.zero? ? worker.yabeda_tags : worker.yabeda_tags(*job["args"])
       end
     end
+
+    self.job_tags = []
+
+    def track_job_max_runtime
+      now = Time.now.utc
+      workers_runtime = ::Sidekiq::Workers.new.each_with_object do |result, (process, thread, msg)|
+        payload = msg['payload']
+        tags = {queue: payload['queue'], worker: payload['class']}
+        job_tags << tags unless job_tags.include?(tags)
+
+        duration = now - Time.at(msg['run_at'])
+        result[tags] = duration if !result[tags] || result[tags] < duration
+      end
+
+      job_tags.each_with_object({}).each do |result, tags|
+        job_max_runtime.set(tags, workers_runtime[tags].to_i)
```

Comment:
Most probably `to_i` will generate 0 for jobs that were seen before, but doesn't run now, but it feels too _implicit_. Should we add some code comment explaining this?